src/doc/using_visit/VisualizationWindows/Interactor_settings.rst

```diff
@@ -21,47 +18,27 @@ that have interactor properties. The **Interactors window** is shown in
 Zoom interactor settings
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-The zoom interactor settings are mostly used when the vis window is in
-zoom mode. When the vis window is in zoom mode, clicking in the vis window
-will anchor a point that becomes one of the corners of a zoom rectangle.
-When you release the mouse, the point over which the mouse was released
-becomes the opposite corner of the zoom rectangle. VisIt's default behavior
-is to show guidelines that extend from the edges of the zoom rectangle to
-the edges of the plots' bounding box when the vis window is in 2D mode. If
-you want to turn off the guidelines, click off the **Show Guidelines** check
-box in the **Interactors window**.
+The zoom interactor settings are mostly used when the vis window is in zoom mode. 
+When the vis window is in zoom mode, clicking in the vis window will anchor a point that becomes one of the corners of a zoom rectangle.
+When you release the mouse, the point over which the mouse was released becomes the opposite corner of the zoom rectangle. 
+VisIt's default behavior is to show guidelines that extend from the edges of the zoom rectangle to the edges of the plots' bounding box when the vis window is in 2D mode. 
+If you want to turn off the guidelines, click off the **Show Guidelines** check box in the **Interactors window**.
 
-When sweeping out a zoom rectangle in zoom mode, VisIt allows you to draw
-a rectangle of any proportion. The relative shape of the zoom rectangle,
-in turn, influences the shape of the viewport drawn in the vis window. If
-you hold down the *Shift* key while sweeping out the zoom rectangle, VisIt
-will constrain the shape of the zoom rectangle to a square. If you want
-VisIt to always force a square zoom rectangle so that you don't have to
-use the *Shift* key, you can click on the **Clamp to Square** check box,
-click **Apply** in the **Interactors window** and save your settings.
+When sweeping out a zoom rectangle in zoom mode, VisIt allows you to draw a rectangle of any proportion. 
+The relative shape of the zoom rectangle, in turn, influences the shape of the viewport drawn in the vis window. 
```

Comment:
I think `aspect ratio` is maybe a better term than `relative shape`. I think we use `aspect ratio` elsewhere both in the GUI and in the docs to mean same thing.